addons/mail/demo/discuss/public_channel_demo.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+
+        <record model="mail.guest" id="mail.public_guest">
+            <field name="name">Guest Alex</field>
```

Comment:
I don't think a guest would name themselves "Guest" if the take time to rename themselves, it would be just Alex probably.

Reply:
but it might be less obvious for the purpose of testing, which is the initial purpose of the PR. prefix Guest indicates that this person is a guest directly.